src/portals/hope/features/claims/claim-details/ClaimCounterparty/InvoiceModal.tsx

```diff
@@ -0,0 +1,218 @@
+import React from 'react'
+import {
+  ThirdLevelHeadline,
+  Modal,
+  Spacing,
+  ButtonsGroup,
+  Button,
+  convertEnumToTitle,
+  lightTheme,
+} from '@hedvig-ui'
+import styled from '@emotion/styled'
+import {
+  RecoveryInvoice,
+  useUpdateRecoveryInvoiceContentMutation,
+} from 'types/generated/graphql'
+import gql from 'graphql-tag'
+import { toast } from 'react-hot-toast'
+import copy from 'copy-to-clipboard'
+
+const InvoiceModalStyled = styled(Modal)`
+  width: 46rem;
+  padding: 15px;
+  background-color: ${({ theme }) => theme.backgroundLight};
+`
+
+const Content = styled.div`
+  width: 100%;
+
+  padding: 15px 100px 25px 15px;
+
+  background-color: ${({ theme }) => theme.background};
+`
+
+const Remark = styled.span`
+  font-size: 14px;
+  color: ${({ theme }) => theme.placeholderColor};
+`
+
+gql`
+  mutation updateRecoveryInvoiceContent(
+    $recoveryInvoiceId: ID!
+    $content: String!
+  ) {
+    recoveryInvoice_updateContent(
+      recoveryInvoiceId: $recoveryInvoiceId
+      content: $content
+    ) {
+      id
+      amount
+      invoiceNumber
+      memberReference
+      dateOfOccurrence
+      recoveryData {
+        recoveryId
+        amount
+        claimType
+      }
+      bankgiroNumber
+      bankAccountNumber
+      paymentDueDate
+      counterparty {
+        id
+        reference
+        insuranceProvider {
+          displayName
+        }
+      }
+    }
+  }
+`
+
+export const InvoiceModal: React.FC<{
+  visible: boolean
+  onClose: () => void
+  invoice: RecoveryInvoice | null
+}> = ({ visible, onClose, invoice }) => {
+  const [updateRecoveryInvoiceContent] =
+    useUpdateRecoveryInvoiceContentMutation()
+
+  const content =
+    invoice === null
+      ? ''
+      : `
```

Comment:
With the change above
```suggestion
  const content = `
```